### ISSUE TITLE
F-Droid

### ISSUE THREAD

**Issue by Mikaela** [ipfs/package-managers#39]
F-Droid is Android package manager and repository for FOSS apps.

@Swedneck seems to have ran a mirror in IPFS, but I am unable to access it currently.

* https://forum.f-droid.org/t/ipfs-mirrors/4489

**Comment by swedneck** [comment]
Yeah sorry about that, it's offline along with a bunch of stuff on my old server, sadly.

**Comment by andrew** [comment]
Added an F-Droid doc here: https://github.com/ipfs/package-managers/commit/5b3211439ede132faad66dcf213abeef28b3fb4a
